generatebundlefile/bundle_types.go

```diff
@@ -96,9 +96,9 @@ type RequiresSpec struct {
 }
 
 type Configuration struct {
-	Name     string `json:"name,omitempty"`
-	Required bool   `json:"required,omitempty"`
-	Default  string `json:"default,omitempty"`
+	Name     string      `json:"name,omitempty"`
+	Required bool        `json:"required,omitempty"`
+	Default  interface{} `json:"default,omitempty"`
```

Comment:
What kind of defaults are giving you problems? Can it be treated like embedded yaml? This interface{} is probably not going to work

Reply:
That `80` doesn't work as a string?

Reply:
There is int and bool in this. https://github.com/aws/eks-anywhere-build-tooling/blob/main/projects/goharbor/harbor/helm/requires-config.yaml#L26

Reply:
those are giving the errors

Reply:
Put quotes around them